.circleci/config.yml

```diff
@@ -18,24 +20,15 @@ jobs:
             pip install sphinx-gallery
             pip install .
             cd doc
-            make html
+            make doctest
       - store_artifacts:
-          path: doc/_build/html
-          destination: doc
+          path: doc/_build/doctest/output.txt
```

Comment:
The previous version was done to to be able to preview built documentation and we still need it.